client/my-sites/themes/single-site-wpcom.jsx

```diff
@@ -39,3 +59,12 @@ export default connectOptions(
 		);
 	}
  );
+
+export default connect(
+	( state, { siteId } ) => {
+		const currentPlan = getCurrentPlan( state, siteId ) || null;
+		return {
+			currentPlanSlug: currentPlan && currentPlan.productSlug,
```

Comment:
Use lodash's `get`:

```js
currentPlanSlug: get( currentPlan, 'productSlug', false )
```

(3rd arg is fallback -- use whatever seems appropriate)